### ISSUE TITLE
List of Issues for 4/19 Meeting

### ISSUE THREAD

**Issue by PhilipMathieu** [PhilipMathieu/unet-orthoimagery#4]
Edit this or add a comment with more ideas
- [x] #6
- [ ] training loss is very noisy even when increasing batch size
- [ ] #15
- [ ] Do we need more raw training data?
- [x] #7
- [x] #8
- [ ] also should figure out how to run batches of parameters either on Colab or Discovery
